public/app/features/alerting/unified/components/rules/central-state-history/EventListSceneObject.tsx

```diff
@@ -39,74 +25,61 @@ import { CollapseToggle } from '../../CollapseToggle';
 import { LogRecord } from '../state-history/common';
 import { useRuleHistoryRecords } from '../state-history/useRuleHistoryRecords';
 
-const LIMIT_EVENTS = 250;
-
-const HistoryEventsList = ({ timeRange }: { timeRange?: TimeRange }) => {
-  const styles = useStyles2(getStyles);
-
-  // Filter state
-  const [eventsFilter, setEventsFilter] = useState('');
-  // form for filter fields
-  const { register, handleSubmit, reset } = useForm({ defaultValues: { query: '' } }); //  form for search field
+import { LABELS_FILTER } from './CentralAlertHistoryScene';
+
```

Comment:
This file includes the following changes:

- Use the new scenes variable for filtering by labels